tests/test_user_views.py

```diff
@@ -157,4 +157,17 @@ def test_map_locations(self):
 
             self.assertEqual(resp.status_code, 200)
 
-            self.assertIn('San Francisco', html)
\ No newline at end of file
+            self.assertIn('San Francisco', html)
```

Comment:
How is this change related to the PR ?

Reply:
I guess this issue is still under review [https://github.com/rtfpessoa/diff2html/issues/157](https://github.com/rtfpessoa/diff2html/issues/157) We shall get back to it later.